src/sentry/eventstream/snuba.py

```diff
@@ -336,24 +336,18 @@ def _send(
 
         data = (self.EVENT_PROTOCOL_VERSION, _type) + extra_data
 
-        # TODO remove this once the unified dataset is available.
-        # Inserting into both events and transactions datasets lets us
-        # simulate what is currently happening via kafka when both the events
-        # and transactions consumers are running.
-        datasets = ["events"]
+        dataset = "events"
```

Comment:
should this be `errors` eventually?

Reply:
No, the name of the dataset is events and did not change.

Reply:
https://github.com/getsentry/snuba/blob/master/snuba/datasets/factory.py#L56